### ISSUE TITLE
Playback Error No compatible streams are currently available. Please try again later or contact your system administrator for details.

### ISSUE THREAD

**Issue by muneebhussain15** [ddurdle/Python-GoogleDrive-VideoStream#129]
I keep getting this error trying to play files.
The strm file generation process works emby downloads thumb nails

however videos dont play

`Playback Error
No compatible streams are currently available. Please try again later or contact your system administrator for details.`
![image](https://user-images.githubusercontent.com/15520829/55684320-42458f00-5963-11e9-8f66-b3f9b70e03cc.png)

**Comment by ddurdle** [comment]
resolved?  normally a good idea typically is to look in the log file generated by the plugin (if you specify to save a log in the settings) and then manually navigate to the url in a browser to confirm if there is really an error.

**Comment by muneebhussain15** [comment]
Hello thanks for following up and yes it was resolved it was a very niche error that is why I closed it.

Basically the reason why it happened was that I using https://ngrok.com service to port forward 9988 instead of using putty.

So when I went to browser console logs I saw

so it turns out it was trying to forward urls from a sub domain of ngrok which didnt exist anymore and hence generating the error.

now restarting the server generation process by using putty port forwarding
